src/devices/Pn532/RfConfiguration/Analog212_424kbpsMode.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Pn532.RfConfiguration
+{
+    /// <summary>
+    /// This CfgItem is used to choose the analog settings that the PN532 will use for the
+    /// baudrates 212/424kbps.
+    /// When using this command, the host controller has to provide 8 values
+    /// (ConfigurationData[]) for the following internal registers
+    /// Actually, there is only one CIU_Demod register which defines a setting used by the
+    /// reader in reception only.But depending on the RF condition, two different settings
+    /// can be used for this register:
+    /// • CIU_Demod when own RF is On defines a setting when its RF field is on
+    /// during a reception i.e.initiator passive mode,
+    /// • CIU_Demod when own RF is Off defines a setting when its RF field is off
+    /// during a reception i.e.initiator active mode.
+    /// Please refer to docuementation for more information
+    /// </summary>
+    public class Analog212_424kbpsMode
+    {
+        public byte CIU_RFCfg { get; set; } = 0x69;
```

Comment:
public class - please expand names

Reply:
I would love to. See documentation page 104+ https://www.nxp.com/docs/en/user-guide/141520.pdf. If you have any way to find out what is the long name for it, that would be great. First one is obvious, will adjust but for the others, it's far from it.

Reply:
https://www.nxp.com/docs/en/nxp/data-sheets/PN532_C1.pdf page 144 seems to have descriptions. Those are not documented in that spec you pointed to (at least I couldn't find it)